spec.yaml

```diff
@@ -355,42 +355,47 @@ paths:
               type: string
           schema:
             $ref: "#/definitions/Error"
-  /invite/accept:
+  invite/forward:
     post:
-      summary: Accept an invite to a shared document
-      description: Foo
+      summary: Forward an invitation to start sharing
+      description: Sends an invitation to an end-user (by mail for example)
```

Comment:
The forward endpoint doesn't send the mail, right? We don't have that functionality that. This endpoint is exposed via a link which would be provided to the user (and in the current case, through the phoenix app), and this just calls /invite/accept on the original user's side (as described [here](https://github.com/sciencemesh/sciencemesh/issues/9)).

Reply:
@ishank011 right, I corrected the wording.